### PR TITLE
Added missing documentation for installing your own production cluster.

### PR DESCRIPTION
**Pull request by beamdee**

### CODE REVIEW

**Comment by sftim** [comment]
@beamdee, if you sign the contributor license agreement we can review this pull request.

/check-cla
(in case you signed already and the automation did not notice)

**Comment by beamdee** [comment]
I signed it

File: content/en/docs/setup/_index.md [hunk 1]
```diff
@@ -112,4 +112,5 @@ The following production environment solutions table lists the providers and the
 | [VMware](https://cloud.vmware.com/) | [VMware Cloud PKS](https://cloud.vmware.com/vmware-cloud-pks)              |[VMware Enterprise PKS](https://cloud.vmware.com/vmware-enterprise-pks)        |   [VMware Enterprise PKS](https://cloud.vmware.com/vmware-enterprise-pks)          | [VMware Essential PKS](https://cloud.vmware.com/vmware-essential-pks)      |          |[VMware Essential PKS](https://cloud.vmware.com/vmware-essential-pks)
 | [Z.A.R.V.I.S.](https://zarvis.ai/) | &#x2714; | | | | | |
 
+If you want to do it yourself, follow the documentation by starting with [installing kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```
Comment (sftim) [inline content/en/docs/setup/_index.md]: Shouldn't this be part of the table above?

Comment (beamdee) [reply content/en/docs/setup/_index.md]: Installing it yourself is a special case wrt to the table so I feel it makes more sense to explicitly mention it.

Comment (sftim) [reply content/en/docs/setup/_index.md]: How about both? Add the kubeadm option to the table _and_ have a section about installing your own production-grade cluster.

Comment (beamdee) [reply content/en/docs/setup/_index.md]: Even better!

**Comment by jimangel** [comment]
@beamdee have you been able to address @sftim's feedback from 2 weeks ago?

File: content/en/docs/setup/_index.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/setup/_index.md]: @beamdee if you add that entry into the table as well, this will be good to merge.

**Comment by zacharysarah** [comment]
@beamdee :wave: Please feel free to `/reopen` this issue when you’re ready to continue.

/close

**Pull request closed without merging**